docs/CHANGELOG.txt

```diff
@@ -23,6 +23,30 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 
 Changelog for PrestaShop 8
 
+####################################
+#   v8.0.4 - (2023-04-13)
+####################################
+
+- Back Office:
+  - Bug fix:
+    - #32028: Fix module active flag with group & shop contexts (by @boherm)
+    - #32028: Fix module active flag with group & shop contexts (by @boherm)
```

Comment:
We have a duplicated line here.

Reply:
Because you deserve that much consideration for your PRs @boherm !!!

Reply:
But actually it should be fixed on 8.0.x branch because it's also duplicated https://github.com/PrestaShop/PrestaShop/blob/8.0.x/docs/CHANGELOG.txt#L32-L33

I suggest not blocking this merge PR for this, and instead fix this on 8.0.x branch, so it will be synced with the next merge PRs